pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -780,6 +780,25 @@ func removeTerminatedContainers(containers []*runtimeapi.Container) []*runtimeap
 func (p *criStatsProvider) addCadvisorContainerStats(
 	cs *statsapi.ContainerStats,
 	caPodStats *cadvisorapiv2.ContainerInfo,
+) {
+        if caPodStats.Spec.HasCustomMetrics {
+                cs.UserDefinedMetrics = cadvisorInfoToUserDefinedMetrics(caPodStats)
```

Comment:
Just curious, why are we also adding un custom cAdvisor metrics here?  Just for completeness?

Reply:
Did you mean this question to `addCadvisorContainerCPUAndMemoryStats` or here?

Reply:
> why are we also adding un custom cAdvisor metrics here? 

Yeah I am basically keeping the same as the existing `addCadvisorContainerStats`.